Add explanation for server and devices not on the same network (#403)
leikoilja/ha-google-home: README.md

If you can see your devices, and they all seem correct, but the alarms/timers sensors do not appear, or appear empty try restarting the Google Home device, Home Assistant, and reinstalling the integration.

### Device offline or not compatible message (#402)

> If you get the following error for all of your devices:

Reviewer: Can we make this bold and italic so it catches peoples eyes?

```suggestion
If you get the following error for ***all*** of your devices:
```

> If you get the following error for ***all*** of your devices:

Reviewer 2: Based on the failed linting: https://github.com/leikoilja/ha-google-home/runs/4373080559?check_suite_focus=true

```suggestion
If you get the following error for **_all_** of your devices:
```

```
2021-11-28 16:13:20 DEBUG (MainThread) [custom_components.google_home] Failed to fetch timers/alarms information from device xxx. We could not determine its IP address, the device is either offline or is not compatible Google Home device. Will try again later.
```

> I may be that your device and Home Assistant's installation are not on the same network.

Reviewer 3: ```suggestion
It may be that your device and Home Assistant installation are not on the same network.
```

> The integration works by connecting to the Google's servers to authenticate and get the authorisation keys for controlling the devices, but after that, all the requests are made locally, so it's required that the server and devices are on the same network. You can use a VPN to overcome this issue.

Reviewer: ```suggestion
The integration works by connecting to the Google's servers to authenticate and get the authorisation keys for controlling these devices, but after that, all the requests are made locally, so it's required that the server and devices are on the same network. You can use a VPN or setup routing between each network to overcome this issue.
```

Reviewer 3: ```suggestion
The integration works by connecting to the Google servers to authenticate and get the authorisation keys for the devices, but after that, all the requests are made locally, so it's required that the server and devices are on the same network. You can use a VPN to overcome this issue.
```

Reviewer: @KapJI I think it is important to include the note i added, that it can also be achieved by setting up routing between each network, in case people are using VLAN's.

Reviewer 3: I'm not against it, my changes are for the current version.
